src/pretix/control/templates/pretixcontrol/event/mail.html

```diff
@@ -92,7 +114,7 @@ <h4 class="panel-title">
                             </a>
                         </h4>
                     </div>
-                    <div id="order_expirew" class="panel-collapse collapse">
+                    <div id="order_expire" class="panel-collapse collapse">
```

Comment:
``order_expirew`` was supposed to be a (apparently bad) abbreviation for "expire *w*arning". If you change it, change the reference in the ``href=`` argument above as well.